Demo/SwiftUI/NamiColorsSWUI/NamiColorsSWUI.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/namiml/NamiSDK-SwiftPackageManager",
         "state": {
           "branch": null,
-          "revision": "bc7f1c3f2697ba929286f1ec5912862bba21ec42",
-          "version": "2.6.0"
+          "revision": "665d18f45267a3805cea41f065e05c66c5b8da23",
+          "version": "2.9.1"
```

Comment:
looks like this will need to be updated before we can merge?

Reply:
Fixed via merge